marconi-chain-index/src/Marconi/ChainIndex/Experimental/Indexers/Utxo.hs

```diff
@@ -93,16 +94,14 @@ data UtxoEvent = UtxoEvent
 $(makeLenses ''UtxoEvent)
 
 type instance Core.Point UtxoEvent = C.ChainPoint
-type instance Core.Point Utxo = C.ChainPoint -- required for the unspent_transaction table SQL mappings
-type instance Core.Point Spent = C.ChainPoint -- required for the spent table SQL mappings
```

Comment:
This is why I wanted to make the change: having to declare a type instance for each type that you need to decorate with a `Point` was tedious.

Reply:
As the `Point` type family still seems to be used in `marconi-core`, how do you map `ChainPoint` to `Point event`? For example, I see `(TimedEvent (Point event) event)` in some signatures.

Reply:
We still have to declare it for `event`, but not for intermediary types that are used internally when we're processing them probably `TimedData` would be a better name for it now.
It allows some simplification in the indexer implementations, [like this one](https://github.com/input-output-hk/marconi/pull/67/files#diff-c07cb25c3219e34c21ad277adaed60d1df737d45eba462fa7ced76487f1b4ef9R174).

Reply:
I understand now. Thanks for the clarification :)

Reply:
Do you think it worths renaming `TimedEvent` to `Timed` or `TimedData` before merging it?